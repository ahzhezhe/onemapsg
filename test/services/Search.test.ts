import { oneMap } from '..';

test('search', async () => {
  const result = await oneMap.search.search({
    searchVal: '542226',
    returnGeom: true,
    getAddrDetails: true
  });
  expect(result).toMatchSnapshot();
});
