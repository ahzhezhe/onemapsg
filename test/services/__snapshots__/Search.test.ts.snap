// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search 1`] = `
{
  "found": 1,
  "pageNum": 1,
  "results": [
    {
      "ADDRESS": "226B COMPASSVALE WALK COMPASSVALE VISTA SINGAPORE 542226",
      "BLK_NO": "226B",
      "BUILDING": "COMPASSVALE VISTA",
      "LATITUDE": "1.38771174427547",
      "LONGITUDE": "103.899302425122",
      "POSTAL": "542226",
      "ROAD_NAME": "COMPASSVALE WALK",
      "SEARCHVAL": "COMPASSVALE VISTA",
      "X": "35343.1499079037",
      "Y": "41071.7320690025",
    },
  ],
  "totalNumPages": 1,
}
`;
